ext/standard/image.c

```diff
@@ -449,7 +449,7 @@ static int php_read_APP(php_stream * stream, unsigned int marker, zval *info)
 	}
 	length -= 2;				/* length includes itself */
 
-	buffer = emalloc(length);
+	buffer = emalloc((zend_long)length);
```

Comment:
`(size_t)` would be better here. (The argument to php_stream_read should also use `(size_t)`.)